frontend/src/components/core/Header/styled-components.ts

```diff
@@ -30,7 +30,12 @@ export const StyledHeader = styled.header<StyledHeaderProps>(
     top: theme.spacing.none,
     left: theme.spacing.none,
     right: theme.spacing.none,
-    height: 0,
+    height: "3.375rem",
+    background: `linear-gradient(0deg, ${transparentize(
+      theme.colors.bgColor,
+      0.5
+    )}, ${theme.colors.bgColor})`,
```

Comment:
Thank you for going the extra mile with this! I can't really speak on which would be the best solution, but I appreciate helping figure it out.

Reply:
No problem! The snapshot situation has been far from ideal for awhile now, but we've been able to put off figuring out how to improve it given how rarely they get updated.

In case these design fixes across the core codebase are likely to be a much more common thing, let us know and we'll try to prioritize improving the process to be much more streamlined